l1-contracts/src/core/libraries/rollup/ValidatorSelectionLib.sol

```diff
@@ -489,6 +501,7 @@ library ValidatorSelectionLib {
     Slot slot = _ts.slotFromTimestamp();
     RollupStore storage rollupStore = STFLib.getStorage();
 
+    // Pending chain tip
     uint256 pendingBlockNumber = STFLib.getEffectivePendingBlockNumber(_ts);
```

Comment:
Was not immediately clear to me that it's the latest block of the pending chain so added a comment here.